R/slack.R

```diff
@@ -66,8 +56,76 @@ slack_data <- function(dat, remote_name, report_url, remote_is_primary) {
            url = report_url)))))
 }
 
+teams_data <- function(dat, remote_name, report_url, remote_is_primary) {
+  content <- prepare_content(dat, remote_name, report_url, "[{label}]({url})")
+  facts <- list(list(name = "id:", value = content$id))
+  if (!is.null(content$git)) {
+    facts <- c(facts, list(list(name = "git:", value = content$git)))
+  }
+  ## Using same colours as slack default for "good" and "warning" messages
+  col <- if (remote_is_primary) "2EB886" else "DAA038"
+
+  list(
+    "@type" = "MessageCard",
+    "@context" = "http://schema.org/extensions",
+    summary = content$fallback,
+    themeColor = col,
+    sections = list(
+      list(
+        activityTitle = content$title,
+        activityText = content$text,
+        ## TODO: Choose another image from somewhere?
```

Comment:
In the orderly feed on slack we use an ambulance? There must be some sort of emoji-as-a-service we could use here?

Reply:
> There's also a failing test that I think you've seen, but just looking over the implementation this seems like a good way to do it given the current constraints on the package. Having two things now will make it easier to get the refactor right I think.
> 
> I don't think we should try for anything too fancy at this point.
> 
> You should bump the version and add a news bullet before too, and if you can remove glue that'd be great.
> 
> Is there a Teams channel with an example you've used in testing? Could you post a a screenshot?

Thanks, fixed that test. Tested this posting to orderly-feed in vimc team. It looks like below atm.

![Selection_046](https://user-images.githubusercontent.com/39248272/79558212-b731c580-809b-11ea-8d37-ba7b0eb6eb2a.png)

I think the connector icon looks pretty terrible but sadly not many options to make this better from teams. You just upload an image and it puts it in a hexagon.

Let me know what you think.

Reply:
I think this is great